### ISSUE TITLE
Run time error

### ISSUE THREAD

**Issue by fulldecent** [gmkou/makeAppIcon#1]
```
› ~/Developer/makeAppIcon/makeAppIcons iTunesArtwork@2x.png out
/Library/Ruby/Site/2.0.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- ./iconFileName (LoadError)
    from /Library/Ruby/Site/2.0.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/williamentriken/Developer/makeAppIcon/makeAppIcons:5:in `<main>'

```

**Comment by gmkou** [comment]
Can't reproduce on may Mac. Please tell me more detailed info. (e.g. environment of your system)

**Comment by fulldecent** [comment]
Please find full transcript and version information attached:

```
Williams-iMac:~ williamentriken$ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.10.3
BuildVersion:   14D136
Williams-iMac:~ williamentriken$ git clone git@github.com:gmkou/makeAppIcon.git
Cloning into 'makeAppIcon'...
remote: Counting objects: 44, done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 44 (delta 18), reused 44 (delta 18), pack-reused 0
Receiving objects: 100% (44/44), 421.65 KiB | 0 bytes/s, done.
Resolving deltas: 100% (18/18), done.
Checking connectivity... done.
Williams-iMac:~ williamentriken$ makeAppIcon/makeAppIcons ~/Desktop/screenshot.png 
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- ./iconFileName (LoadError)
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from makeAppIcon/makeAppIcons:5:in `<main>'
Williams-iMac:~ williamentriken$ 
```

**Comment by sassy** [comment]
@fulldecent 
fail to resolve relative path.
you should not run it in parent directory.
you will try following.

```
$ cd makeAppIcon
$ ./makeAppIcons --in ~/Desktop/screenshot.png --out ./out
```

**Comment by sassy** [comment]
@gmkou 
you should use `require_relative`.

**Comment by gmkou** [comment]
@sassy  Thanks.

**Comment by gmkou** [comment]
@fulldecent Could you try this branch?
https://github.com/gmkou/makeAppIcon/tree/work-1-require

**Comment by fulldecent** [comment]
Yes, this works. Thank you

**Comment by gmkou** [comment]
Good.

**Comment by gmkou** [comment]
Fix by pull request #2
